website/content/docs/datasources/http/http.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
The build is failing because the file type should be `mdx`. 

The best place to put this doc is directly under the `datasources` folder, as `datasources/http.mdx` because it's a single datasource type.